mailpoet/assets/js/src/landingpage/content.tsx

```diff
@@ -0,0 +1,164 @@
+import { __ } from '@wordpress/i18n';
+import { Heading } from 'common/typography/heading/heading';
+import { Grid } from 'common/grid';
+
+type LandingpageImages = {
+  heroImage: string;
+  featureImages: {
+    icon_1: string;
+    icon_2: string;
+    icon_3: string;
+  };
+  wooCommerceFeatureImages: {
+    feature_1: string;
+    feature_2: string;
+    feature_3: string;
+  };
+};
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const Images: LandingpageImages = window.mailpoet_landingpage_images;
+
+function Content() {
+  return (
+    <div className="landing-content">
+      <div className="hero-section mailpoet-content-center">
```

Comment:
@triple0t what do you think about using some semantic HTML? like sections for the hero, general features and FAQs? Maybe also lists on the features?